src/Web/Avalonia.Web.Blazor/Avalonia.Web.Blazor.csproj

```diff
@@ -36,11 +37,11 @@
     <AvaloniaResource Include="Assets\*" />
     <Content Include="*.props">
       <Pack>true</Pack>
-      <PackagePath>build\;buildTransitive\</PackagePath>
+      <PackagePath>build\</PackagePath>
     </Content>
     <Content Include="*.targets">
       <Pack>true</Pack>
-      <PackagePath>build\;buildTransitive\</PackagePath>
```

Comment:
Keep .targets in buildTransitive too. We are removing .props from there because Blazor SDK generates buildTransitive\<PackageName>.props for us. It doesn't do that for .targets

Reply:
fixed.